Makefile

```
@@ -31,9 +31,10 @@ endif
 	rm -f ${RESOURCE_YAML} args.yaml
   # Copy src templates
 	cp src/templates/* ${CHART_DIR}/templates
-ifneq ($(CHART_VERSION),latest)
-	sed -i "" -e "s/^appVersion:.*/appVersion: ${CHART_VERSION}/" ${CHART_DIR}/Chart.yaml
-endif
+
+fetch: prep
+  # Set value in Chart.yaml
+	yq eval -i '.appVersion = "$(shell yq eval '.data.version | sub("v"; "")' ${CHART_DIR}/templates/dashboard-info-cm.yaml)"' ${CHART_DIR}/Chart.yaml
```

I think this would be cleaner

```suggestion
	yq -i '.appVersion = (load("$(CHART_DIR)/templates/dashboard-info-cm.yaml").data.version | sub("v"; ""))' Chart.yaml
```

---

Hi Marten.. I tried a few ways of updating Chart.yaml with nested conditionals (to decipher the OS) inside the existing ifneq. That pushed me to change direction and add the new target with a dependency.

Thanks for the suggestion - your cmd actually works as part of the original target "fetch".. I have reverted some of the code and kept your version of the yq cmd.

Pushing again - cheers!